User Engagement Lead/survey_questions.md

```diff
@@ -0,0 +1,80 @@
+- It covers different aspects of ESMValTool like application, usage, documentation and community
+
+The survey can be implemented in a survey app like surveymonkey that does not
+collect personal data. The owner is “user engagement team”. They will
+edit/run/evaluate the survey and communicate the results to other teams if
+needed.
+
+## Survey content
+
+Hi! We in [ESMValGroup](https://www.esmvaltool.org/), would love to hear about
+your research and experiences using ESMValTool. Here, there are a few questions
+which help to create your story. Your answers will help us to improve ESMValTool
+and hence enhance the science that you conduct. All responses are recorded
+anonymously. Let’s begin:
+
+**Describe your research/case study and challenges:**
+
+1. Can you describe your research/case study?
+2. Can you underline the challenges in your research/case study?
+3. How would you categorize your expertise level in using ESMValTool? (Beginner,
+   competent practitioner, expert)
+
+**Software:**
+
+ESMValTool provides many features and functions to analyze data for Earth system
+models.
+[Here](https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/preprocessor.html#preprocessor)
+you can find a list of preprocessors.
+
+4. Which features/functions in ESMValTool do you think are most useful for your
+research/case study?
+5. How satisfied are you with the software's:
+    - Ease of use (1-10)
+    - Development workflow (1-10)
+    - Reliability (1-10)
+
+**Documentation and tutorial:**
+
+ESMValTool [documentation](https://docs.esmvaltool.org/en/latest/) covers
+different aspects of research: How the code work, how to obtain and process
+data, and what are best practices in developing code. There is also an online
+[tutorial](https://esmvalgroup.github.io/ESMValTool_Tutorial/) that shows not
+only how to run an experiment but also how to develop your own code.
+If you are familiar with documentation and tutorial, please tell us:
+
+6. What are the most useful parts?
+7. What topics are missing that you want to learn about?
+8. How much do you agree with the following statements?
+    - It is easy to find relevant information from the documentation. (1-10)
+    - The instructions in the documentation and tutorial are complete and easy
+      to follow. (1-10)
+    - I can immediately apply what I learned from the tutorial. (1-10)
+
+**Community:**
+
+ESMValTool provides [different
+platforms](https://docs.esmvaltool.org/en/latest/introduction.html#support) to
+communicate with users and researchers: mailing list, github issues, discussion
+page.
+
+9. If you have used any of these, please tell us how much do you agree with
+the following statements?
+    - I felt comfortable asking my questions using these platforms. (1-10)
+    - I was able to get clear answers to my questions from the community. (1-10)
+10. What suggestions do you have for improvements, or any other relevant input for us?
+
```

Comment:
Is the last question tailored towards only the "community" part, or does it refer to any other suggestions in general? I think it should be the latter and the question could be asked without a number, just a general "any other comments" kind of thing. 
We would then have the 10. question free for another community question. We could for example ask how many of the different ways there are to connect with the community were known by the person answering the questionnaire, e.g. a bullet list with each of the items (mailing list, github issues, discussion page, webpage, twitter...) where multiple answers are possible.

Reply:
I agree with this idea.